SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -262,11 +247,15 @@ void USpatialGameInstance::HandleOnPlayerSpawnFailed(const FString& Reason)
 	OnSpatialPlayerSpawnFailed.Broadcast(Reason);
 }
 
-void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel, UWorld* OwningWorld)
+void USpatialGameInstance::OnLevelInitializedNetworkActors(ULevel* LoadedLevel, UWorld* OwningWorld) const
 {
+	UE_LOG(LogSpatialOSNetDriver, Log, TEXT("OnLevelInitializedNetworkActors: Level (%s) OwningWorld (%s) World (%s)"),
+		*GetNameSafe(LoadedLevel), *GetNameSafe(OwningWorld), *GetNameSafe(OwningWorld));
+
 	if (OwningWorld != GetWorld()
 		|| !OwningWorld->IsServer()
-		|| !GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking()
+		|| OwningWorld->GetNetDriver() == nullptr
+		|| !Cast<USpatialNetDriver>(OwningWorld->GetNetDriver())->IsReady()
```

Comment:
What if we don't have a spatial enabled?

Reply:
Then this delegate is never bound, it's inside a `if (HasSpatialNetDriver())` conditional - I can add something here if you want extra future security?

Reply:
No that's fine - was just concerned that we had this check in previously